bin/anthology/utils.py

```diff
@@ -227,11 +227,13 @@ def remove_extra_whitespace(text):
     return re.sub(" +", " ", text.replace("\n", "").strip())
 
 
-def infer_url(filename, prefix=data.ANTHOLOGY_PREFIX):
-    """If URL is relative, return the full Anthology URL."""
+def infer_url(filename, template=data.CANONICAL_URL_TEMPLATE):
```

Comment:
Have you checked that using the canonical URL is the correct behaviour for all existing `infer_url` calls? On a glance I'd say yes, but I haven't thoroughly checked

Reply:
Yes, I went through the calls to infer_url.